pandas/tests/indexing/test_categorical.py

```diff
@@ -261,10 +263,14 @@ def test_loc_listlike(self):
         expected = self.df.iloc[[4, 0, 1, 5]]
         tm.assert_frame_equal(result, expected, check_index_type=True)
 
```

Comment:
can you split the errors conditions to a separate test

Reply:
good idea, will update

Reply:
updated + green